roles/common/tasks/main.yml

```diff
@@ -41,18 +48,8 @@
 - name: Upgrade APT to the lastest packages
   apt: upgrade=full
 - name: Rebooting machine so new kernel is active
-  command: shutdown -r now removes=/var/run/reboot-required
-  async: 0
-  poll: 0
-  ignore_errors: true
-  register: restarted
-- name: Waiting for reboot...
-  local_action: wait_for host={{ ansible_ssh_host | default(inventory_hostname) }} search_regex=OpenSSH port=22 delay=15 state=started
-  sudo: no
-  when: restarted.changed
-- name: Rebooting machine so new kernel is active
-  command: shutdown -r now removes=/var/run/reboot-required
-  async: 0
+  command: sleep5 && shutdown -r now removes=/var/run/reboot-required
```

Comment:
Fixed in:
https://github.com/nlesc-sherlock/emma/commit/f1f4a6ad82374a69e7d6e505136fc850009d7742